package.json

```diff
@@ -6,14 +6,14 @@
   "dependencies": {
     "@emotion/react": "11.10.5",
     "@emotion/styled": "11.10.5",
-    "@graasp/chatbox": "github:graasp/graasp-chatbox#v1.0.0",
-    "@graasp/query-client": "github:graasp/graasp-query-client.git",
-    "@graasp/sdk": "github:graasp/graasp-sdk.git#60-pdf-link",
-    "@graasp/translations": "github:graasp/graasp-translations.git",
-    "@graasp/ui": "github:graasp/graasp-ui.git#v0.4.1",
-    "@mui/icons-material": "5.10.9",
+    "@graasp/chatbox": "1.0.0",
+    "@graasp/query-client": "github:graasp/graasp-query-client",
```

Comment:
Should we setup release-please versions on query client?

Reply:
Actually release-please is already setup, but we need to publish the first version to NPM. See https://github.com/graasp/graasp-query-client/issues/221